core/controller/src/main/scala/whisk/core/loadBalancer/SharedDataService.scala

```diff
@@ -75,6 +53,7 @@ class SharedDataService(storageName: String) extends Actor with ActorLogging {
    * Subscribe this node for the changes in the Map, initialize the Map
    */
   override def preStart(): Unit = {
+    node.joinSeedNodes(seedNodes)
```

Comment:
I'm a little unsure about `joinSeedNodes` to be used from within an actor. IIUC this command has a cluster wide impact, it's not only related to this actor, in which case, wouldn't it make sense to surface it in a more global place ?